nowcasting_dataset/config/model.py

```diff
@@ -115,10 +115,34 @@ class Satellite(DataSourceMixin):
         description="The path which holds the satellite zarr.",
     )
     satellite_channels: tuple = Field(
-        SAT_VARIABLE_NAMES, description="the satellite channels that are used"
+        SAT_VARIABLE_NAMES[1:], description="the satellite channels that are used"
     )
-    satellite_image_size_pixels: int = IMAGE_SIZE_PIXELS_FIELD
-    satellite_meters_per_pixel: int = METERS_PER_PIXEL_FIELD
+    satellite_image_size_pixels: int = Field(
+        IMAGE_SIZE_PIXELS_FIELD.default // 3,
+        description="The number of pixels of the region of interest for non-HRV satellite "
+        "channels.",
+    )
+    satellite_meters_per_pixel: int = Field(
+        METERS_PER_PIXEL_FIELD.default * 3,
+        description="The number of meters per pixel for " "non-HRV satellite channels.",
```

Comment:
Tiny nit (almost certainly introduced by Black!)

```suggestion
        description="The number of meters per pixel for non-HRV satellite channels.",
```